src/hotspot/cpu/x86/x86.ad

```diff
@@ -7818,7 +7818,7 @@ instruct vblendvpI(legVec dst, legVec src1, legVec src2, legVec mask) %{
 %}
 
 instruct vblendvpFD(legVec dst, legVec src1, legVec src2, legVec mask) %{
-  predicate(UseAVX > 0 &&
+  predicate(UseAVX > 0 && !EnableX86ECoreOpts &&
```

Comment:
Why do you not call newly added macro assembly routine from its instruction encoding, we already have EnableX86ECoreOpts checks within it.

Reply:
I would have to change the effect to give myself a temp for the scratch register.. Seems safer not to change existing behaviour.

Reply:
That should be fine, lets not penalize getting perf improvement due to additional temp.